src/Components/Facility/Consultations/components/LinePlot.tsx

```diff
@@ -102,19 +102,27 @@ export const LinePlot = (props: any) => {
         text: "",
       },
       grid: {
-        left: "15px",
-        right: "30px",
+        show: false,
+        borderColor: "transparent",
+        left: "40px",
+        right: "20px",
```

Comment:
@skks1212 changing LinePlot is causing side effects in summary plots
![image](https://user-images.githubusercontent.com/29787772/200872219-e96bb214-ac80-4fb8-a467-4adb77e58bee.png)

Reply:
Looks like a result of an earlier PR. Il fix it asap.